src/main/java/gov/cabinetofice/gapuserservice/web/SpotlightController.java

```diff
@@ -119,4 +119,44 @@ public RedirectView callback(@RequestParam String code, @RequestParam String sta
     }
 
 
+    @GetMapping("/oauth/refresh")
+    public ResponseEntity refresh(final HttpServletRequest httpRequest) throws Exception {
+        log.info("SpotlightController /oauth/refresh");
+
+        if (!roleService.isSuperAdmin(httpRequest)) {
```

Comment:
We may be able to use:

`@PreAuthorize("hasRole('SUPER_ADMIN')")`

Reply:
This didn't work, it resulted in the URL always being Access Denied. The existing implementation is consistent with the rest of the user service controllers, but I do agree as tech debt we should look at using those @PreAuthorize annotations to control access